client/src/test/java/com/tencent/rss/client/impl/CheckBlockIdsBenchmark.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Tencent is pleased to support the open source community by making
+ * Firestorm-Spark remote shuffle server available.
+ *
+ * Copyright (C) 2021 THL A29 Limited, a Tencent company.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * https://opensource.org/licenses/Apache-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package com.tencent.rss.client.impl;
+
+import com.tencent.rss.common.util.RssUtils;
+import com.tencent.rss.storage.HdfsTestBase;
+
+import java.io.IOException;
+import java.util.Iterator;
+import java.util.concurrent.atomic.AtomicLong;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.roaringbitmap.longlong.Roaring64NavigableMap;
+import com.tencent.rss.client.util.ClientUtils;
+
+public class CheckBlockIdsBenchmark {
```

Comment:
It may not be appropriate to add a benchmark in the ut package especially for this single operation's benchmark. You could just add the conclusion in your normal codes comments.

Reply:
Spark seems to contain some benchmarks in code...

Reply:
I think microbenchmark is ok in the code. But our code style is not put benchmark into our code. We should keep consistent with the former code.

Reply:
Removed benchmark...